docsrc/assets/setup-postfix.rst

```diff
@@ -23,7 +23,13 @@ server and engineer delivery via LMTP.  The following examples show the
 1.  Setup your recipient maps, thus defining for which recipients the
     ``virtual`` destination will be used::
 
-        postconf -e "virtual_recipient_maps=hash:/etc/postfix/virtual_recipient_domains,hash:/etc/postfix/virtual_recipients"
+        postconf -e "virtual_mailbox_domains=hash:/etc/postfix/virtual_recipient_domains"
+        postconf -e "virtual_mailbox_maps=hash:/etc/postfix/virtual_recipients"
+
+    or, if you have enabled smmapd you can automatically track mailboxes with::
+
+        postconf -e "virtual_mailbox_domains=hash:/etc/postfix/virtual_recipient_domains"
+        postconf -e "virtual_mailbox_maps=socketmap:unix:/run/cyrus/socket/smmap"
```

Comment:
Should this be made consistent with "/var/run/cyrus/socket/smmap" in smmapd.8?

Reply:
I debated this, and decided to leave it consistent within the setup document instead.

lmtp is /run/cyrus/socket/lmtp in this file although /var/imap/socket/lmtp in all others.

Ideally these paths are consistent across all files. It seems to me that distributions are moving towards /run instead of /var for daemon sockets, so users reading this document will probably find the socket as stated, however all the rst files and example cyrus.conf files are written the other way.

Do you have a preference as to which path to use?

Reply:
> Do you have a preference as to which path to use?

Not particularly... I think staying consistent within the present inconsistency is probably the right thing to do for now!